src/staff-handbook/when-and-where-you-work.md

```diff
@@ -7,8 +7,7 @@ permalink: /:path/:basename/
 
 Our working hours are 10:00-18:00, Monday to Friday. Some people work different
 hours by arrangement. Anyone is free to do that as long as their hours of work
-don't make it hard for other people to get things done. For example, many people
-arrive earlier than 10:00 and leave earlier, which is generally fine.
+don't make it hard for other people to get things done. For example, many people start earlier than 10:00 and leave earlier, which is generally fine.
```

Comment:
I like the change from `arrive` to `start` - should the same be done with `leave`, changing it to `finish`?